Chap_API_Server.tex

```diff
@@ -4108,10 +4108,10 @@ \subsubsection{Group Operation Constants}
 
 \begin{constantdesc}
 %
```

Comment:
These values appear to only be used between library and RM? If so, we could decide we don't want to specify them?

Reply:
see above